recipes/gimmemotifs/meta.yaml

```diff
@@ -9,7 +9,7 @@ source:
   sha256: 917259c8a52ab8ea5c877e9b43f45e077b0ceb9a151ca52791fae67ee8689c0d 
 
 build:
-  number: 0
+  number: 1 
   skip: True  # [not py36]
```

Comment:
Note that we'll probably drop building for Python 3.6 by default in a couple of months.